src/include/sof/audio/generic_processor/hifi_codec/codec.h

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Marcin Rajwa <marcin.rajwa@linux.intel.com>
+ */
+
+#ifndef __SOF_AUDIO_HIFI_CODEC__
+#define __SOF_AUDIO_HIFI_CODEC__
+
+#include <sof/audio/generic_processor/hifi_codec/public.h>
+
+
+/*****************************************************************************/
+/* Codec private data structures					     */
+/*****************************************************************************/
+
+struct hifi_codec_config {
+	size_t size;
+	bool avail;
+	void *data; /* tlv config */
+};
+
+struct hifi_codec_data {
+	struct hifi_codec_config s_cfg;
+	struct hifi_codec_config r_cfg;
+};
```

Comment:
What does s_cfg / r_cfg means? At least we should add some comments unless we find a better name.

Reply:
s_cfg and r_cfg mean setup_config and runtime_config respectively.

Reply:
@mrajwa Use the full names, it won't break any line lengths (especially considering limit is now 100)